extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -364,6 +379,15 @@ describe('amp-a4a', () => {
           expect(xhrMock).to.be.calledOnce;
         });
       });
+
+      it('should be able to create AmpContext', () => {
+        return a4a.layoutCallback().then(() => {
+          const window_ = a4aElement.childNodes[0].contentWindow;
```

Comment:
Probably safer to use a `querySelector` to get reference to the `iframe`, so that this test doesn't break if we add stuff to or rearrange the DOM structure under the a4a element.